.bacon.yml

```diff
@@ -146,3 +146,17 @@ test_suites:
     script_env: {}
     criteria: MERGE
     queue_name: small
+  - name: vrt-v3
+    script_path: /root/okta/okta-signin-widget/scripts
+    sort_order: '32'
+    timeout: '40'
+    script_name: vrt-v3
+    criteria: MERGE
+    queue_name: small
+  - name: vrt-v3-get-screenshots
+    script_path: /root/okta/okta-signin-widget/scripts
+    sort_order: '33'
+    timeout: '40'
```

Comment:
Do these suites actually take this long? If not, we should set them to something closer to the usual run time.

Reply:
yeah no this should be changed to about 15 mins